src/xcode/ENA/ENA/Source/Client/__tests__/Mocks/ClientMock.swift

```diff
@@ -59,7 +63,12 @@ extension ClientMock: Client {
 	}
 
 	func getTestResult(forDevice device: String, completion completeWith: @escaping TestResultHandler) {
-		completeWith(.success(2))
+		guard let onGetTestResult = self.onGetTestResult else {
+			completeWith(.success(2))
```

Comment:
What does the magic number `2 `mean?

Reply:
2 indicates a positive test result. I introduced a new variable to make this more clear.